.github/workflows/label_conflicts.yml

```diff
@@ -0,0 +1,20 @@
+name: Auto Label Conflicts
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [master]
+
+jobs:
+  auto-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: prince-chrismc/label-merge-conflicts-action@v1
+        with:
+          conflict_label_name: "has conflicts"
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You can use the @conan-center-bot's PAT if you'd like it to apply the labels

Reply:
https://github.com/prince-chrismc/label-merge-conflicts-action#faq---how-do-i-fix-resource-not-accessible-by-integration

This needs to be changes.